xonsh/execer.py

```diff
@@ -8,7 +8,7 @@
 import warnings
 from collections import Mapping
 
-from xonsh import ast
+from xonsh.ast import CtxAwareTransformer
```

Comment:
In most places you've changed the import to be more explicit -- any reason why that's flipped here?